File: Bind75/1732/Solution.js
/**
 * @param {number[]} gain
 * @return {number}
 */
var largestAltitude = function(gain) {
    let highestAltitude =0;
    let current =0;
    for(let i =0;i< gain.length;i++){
        current +=gain[i];
        highestAltitude =Math.max(highestAltitude,current);
    }
    return highestAltitude;
};

File: Bind75/1732/Solution.java

class Solution {
    public int largestAltitude(int[] gain) {
        int highestAltitude =0;
        int current =0;
        for(int i =0;i<gain.length;i++){
            current +=gain[i];
            if(current>highestAltitude){
                highestAltitude =current;
            }
        }
        return highestAltitude;
    }
}